### PR TITLE
docs(README.md): add nice badges to README

### PR DESCRIPTION
**Pull request by jianan1104**
# Overview

add nice badges to README

# Details
image

### CODE REVIEW

**Comment by odashi** [comment]
I'm a bit thinking if it's better that the badges are placed on the header (before the title) or not:

無題

**Comment by jianan1104** [comment]
> I'm a bit thinking if it's better that the badges are placed on the header (before the title) or not. WDYT
> 
> 無題

I think it's okay for any style. most github repos are let badges under the title. btw, I think we need a pretty avatar or cover
image

**Comment by odashi** [comment]
Great, I have no strong opinions and it'd be okay to go with the most common way.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# `latexify`
+# latexify
+[![Python](https://img.shields.io/pypi/pyversions/latexify-py.svg?style=plastic)](https://badge.fury.io/py/latexify-py)
```
Comment (odashi) [inline README.md:2]: The fury.io address actually points to the PyPI. We don't need to add additional third-party address.

```suggestion
[![Python](https://img.shields.io/pypi/pyversions/latexify-py.svg)](https://pypi.org/project/latexify-py/)
```

**Comment by jianan1104** [comment]
> Great, I have no strong opinions and it'd be okay to go with the most common way.

Sure, let's do it.

**Review by odashi — APPROVED** [review]

**Pull request merged**